tests/topotests/bgp_tcp_mss/test_bgp_tcp_mss.py

```diff
@@ -142,6 +152,12 @@ def _bgp_check_neighbor_tcp_mss(router, neigh):
     assert (
         result is None
     ), 'Failed to sync TCP MSS value over BGP session in "{}"'.format(router1.name)
+
+    test_func = functools.partial(_bgp_check_neighbor_tcp_mss, router1, "192.168.254.2")
+    success, result = topotest.run_and_expect(test_func, None, count=3, wait=0.5)
```

Comment:
Please use higher timeouts, this will 100% fail under the loaded system. Minimum is 15 seconds (count=15, wait=1).

Reply:
> Please use higher timeouts, this will 100% fail under the loaded system. Minimum is 15 seconds (count=15, wait=1).

Fixed